test/PolyhedralGeometry/cone.jl

```diff
@@ -1,178 +1,179 @@
 const pm = Polymake
+let
+    NF, sr2 = quadratic_field(2)
+    Qx, x = QQ["x"]
+    K, (a1, a2) = embedded_number_field([x^2 - 2, x^3 - 5], [(0, 2), (0, 2)])
```

Comment:
I think this should all just be inside a top level `@testset` (probably including the definition of `pm`.

As a variant, it could also be a `@testset ... for` like so (I would need to but can't modify the next couple lines, too, but I hope the idea is clear):

```suggestion
@testset "Cone{$(elem_type(f))} over $f" for f, _ in [
        quadratic_field(2),
        QQ["x"],
        embedded_number_field([x^2 - 2, x^3 - 5], [(0, 2), (0, 2)]),
        ]
```